Makefile

```diff
@@ -178,6 +183,13 @@ bin/crio.cross.%: git-vars .gopathok .explicit_phony
 crioimage: git-vars
 	$(CONTAINER_RUNTIME) build -t ${CRIO_IMAGE} .
 
+local-image:
+	$(TESTIMAGE_SCRIPT)
+
+test-images:
+	$(TESTIMAGE_SCRIPT) -g 1.13 -a amd64
+	$(TESTIMAGE_SCRIPT) -g 1.10 -a amd64
```

Comment:
Wonder why are we using golang 1.13 and 1.10 here?

Asking since I'm trying to test 1.16 branch with newer runc, and it chokes on golang 1.10.

Reply:
@saschagrunert @haircommander PTAL ^^^

Reply:
I feel like this was a version that was in debian or some slow moving distro, but I forget exactly.

Reply:
We had a backwards compatibility to go 1.10 on 1.16.x. Since it is now highly EOL we can consider dropping it. Unfortunately we would need kind of a policy how to handle golang versions on release branches, which would further increase the maintenance efforts on our side.